c/src/point_one/polaris/polaris.c

```diff
@@ -421,6 +483,12 @@ int Polaris_SendECEFPosition(PolarisContext_t* context, double x_m, double y_m,
     P1_Print("Error: Polaris connection not currently open.\n");
     return POLARIS_SOCKET_ERROR;
   }
+#ifdef POLARIS_USE_TLS
+  else if (context->ssl_ctx == NULL || context->ssl == NULL) {
+    P1_Print("Error: Polaris SSL connection not established.\n");
```

Comment:
Is this actually accurate as the error? I kind of wonder if we should just say the SSL.context is unavailable